e2e/test/scenarios/question/offset.cy.spec.ts

```diff
@@ -503,9 +554,523 @@ describe("scenarios > question > offset", () => {
+        const customColumnName = "CC Product Category";
+        const query: StructuredQuery = {
+          "source-table": ORDERS_ID,
+          expressions: {
+            [customColumnName]: [
+              "field",
+              PRODUCTS.CATEGORY,
+              { "base-type": "type/Text", "source-field": ORDERS.PRODUCT_ID },
+            ],
+          },
+          aggregation: [SUM_TOTAL_AGGREGATION, OFFSET_SUM_TOTAL_AGGREGATION],
+          breakout: [["expression", customColumnName], BREAKOUT_DATETIME],
+          "order-by": [["asc", BREAKOUT_DATETIME]],
+        };
+
+        createQuestion({ query }, { visitQuestion: true });
+
+        verifyNoQuestionError();
+        verifyTableContent([
+          ["Doohickey", "2022", "9,031.56", ""],
+          ["Gadget", "2022", "10,672.63", "9,031.56"],
+          ["Gizmo", "2022", "9,929.32", "10,672.63"],
+        ]);
+
+        openNotebook();
+        getNotebookStep("summarize").icon("play").should("be.visible");
+      });
+
+      it("works when custom column is a simple expression", () => {
+        const customColumnName = "1 + 1";
+        const query: StructuredQuery = {
+          "source-table": ORDERS_ID,
+          expressions: {
+            [customColumnName]: ["+", "1", "1"],
+          },
+          aggregation: [SUM_TOTAL_AGGREGATION, OFFSET_SUM_TOTAL_AGGREGATION],
+          breakout: [["expression", customColumnName], BREAKOUT_DATETIME],
+          "order-by": [["asc", BREAKOUT_DATETIME]],
+        };
+
+        createQuestion({ query }, { visitQuestion: true });
+
+        verifyNoQuestionError();
+        verifyTableContent([
+          ["2", "2022", "42,156.87", ""],
+          ["2", "2023", "205,256.02", ""],
+          ["2", "2024", "510,045.03", ""],
+        ]);
+
+        openNotebook();
+        getNotebookStep("summarize").icon("play").should("be.visible");
+      });
+    });
+
+    describe("when custom column is not in the first place of breakout", () => {
+      it("works with custom column that contains a function", () => {
+        const customColumnName = "CC Product Category";
+        const query: StructuredQuery = {
+          "source-table": ORDERS_ID,
+          expressions: {
+            [customColumnName]: [
+              "concat",
+              [
+                "field",
+                PRODUCTS.CATEGORY,
+                {
+                  "base-type": "type/Text",
+                  "source-field": ORDERS.PRODUCT_ID,
+                },
+              ],
+              " from products",
+            ],
+          },
+          aggregation: [SUM_TOTAL_AGGREGATION, OFFSET_SUM_TOTAL_AGGREGATION],
+          breakout: [BREAKOUT_DATETIME, ["expression", customColumnName]],
+          "order-by": [["asc", BREAKOUT_DATETIME]],
+        };
+
+        createQuestion({ query }, { visitQuestion: true });
+
+        verifyNoQuestionError();
+        verifyTableContent([
+          // TODO: should there be "prev" values?
```

Comment:
I believe yes, and in other cases too

Reply:
no, they are valid everywhere, [context](https://metaboat.slack.com/archives/C0645JP1W81/p1726133144398459?thread_ts=1725997000.451069&cid=C0645JP1W81)